### DISCUSSION TITLE
Load weights on a Conv layer

### DISCUSSION THREAD

**Discussion by mattiadutto** [google/flax#2710] [Q&A]
Hi, I'm new to the framework and I would like to ask if is possible to do something like this:
`self.conv.weight = nn.Parameter(torch.from_numpy(array))`
This was done in pytorch.
I'm considering to replace pytorch with flax for improving the speed of the ML project. 
Thank you.

**Comment by marcvanzee** [comment]
Hello @mattiadutto, yes, we also have neural network parameters in Flax and you can use JAX numpy arrays to specify the weights. A JAX numpy is defined using `jax.numpy.array(...)`.

Perhaps it is a good idea to start with some of our beginner material so you can an idea of how our library works, e.g. [Getting started](https://flax.readthedocs.io/en/latest/getting_started.html) or [Flax Basics](https://flax.readthedocs.io/en/latest/guides/flax_basics.html).

**Reply by mattiadutto** [reply]
Hi @marcvanzee, thanks for the answer, I will give you a look to your suggestions
